src/geometry/WorkingGeometry.ts

```diff
@@ -38,4 +38,25 @@ export class WorkingGeometry {
      * must be equal to 0 to ensure that it is a point in Affine space.
      */
     public controlPoints: vec4[];
+
+    /**
+     * Return the BakedGeometry representing the current state of the WorkingGeomtry.
+     *
+     * @returns {BakedGeometry}
+     */
+    public bake() {
```

Comment:
Just one nitpick, could you please add the return type for this function? 

i.e., 

```typescript
public bake(): BakedGeometry {
    // .. .
}
```

Reply:
Added. Do you know if there's a way to enforce this via the `linter`?